components/createNft/FileUploadSection.tsx

```diff
@@ -0,0 +1,211 @@
+      )}
+
+      <label
+        onDragEnter={handleDrag}
+        onDragLeave={handleDrag}
+        onDragOver={handleDrag}
+        onDrop={handleDrop}
+        htmlFor="input-file-upload"
+        className={classNames(
+          'h-72 px-20 border-black border-dashed w-full rounded-xl border-2 flex flex-col cursor-pointer items-center justify-center',
+          dragActive ? 'bg-black/20' : '',
+        )}
+      >
+        <input
+          ref={inputFileRef}
+          accept="image/jpeg, image/png, image/gif, video/mp4, video/quicktime, model/gltf-binary            "
+          type="file"
+          id="input-file-upload"
+          multiple
+          className="hidden"
+          onChange={handleChange}
+        />
+
+        <UploadFile />
+        <p className="my-5 text-sm text-center text-gray-400">
+          Click to browse or
+          <br />
+          drag and drop your files.
+        </p>
+        <Button
+          title="Add Files"
+          className="bg-black text-white w-full"
+          onClick={onButtonClick}
+        />
+      </label>
+      <ul className="text-sm mt-6 list-disc px-[5%] ">
+        <li>Up to 2 files and 100mb in total size</li>
+        <li>
+          <div>
+            <span> Support of image, video and 3D files</span>
+            <Tooltip className="ml-5">
+              <section className="text-xs whitespace-nowrap bg-white">
+                <p className="pb-1 border-b p-3 border-black font-semibold">
+                  File Types
+                </p>
+                <ul className="p-3 space-y-3">
+                  <li>
+                    <div>
+                      <p className="font-semibold">Images</p>
+                      <p className="text-gray-400">JPEG / PNG / GIF</p>
+                    </div>
+                  </li>
+                  <li>
+                    <div>
+                      <p className="font-semibold">Videos</p>
+                      <p className="text-gray-400">MP4 / MOV</p>
+                    </div>
+                  </li>
+                  <li>
+                    <div>
+                      <p className="font-semibold">3D</p>
+                      <p className="text-gray-400">GLB</p>
+                    </div>
+                  </li>
+                </ul>
+              </section>
+            </Tooltip>
+          </div>
+        </li>
+      </ul>
+      <section className="my-8 space-y-2 text-sm">
+        <h4 className="font-bold ">Your Files</h4>
+        <p>
+          The files will be minted in your NFT. The order of the files will be
+          displayed by the order of the NFTs on your files. The NFT can include
+          up to 2 files, including the thumbnail.
+        </p>
+      </section>
+
+      <section className=" space-y-3 text-sm mb-14 max-w-full ">
+        <p className="font-bold">Thumbnail</p>
+        <p>The thumbnail needs to be an image.</p>
+
+        {formData?.nftFiles?.map((file: File, index: number) => (
```

Comment:
Files should be selectable as either a thumbnail or a normal image. Currently this is not handled in any way and all files are added under "thumbnails".

Reply:
I did not get this point.

Reply:
Please take a look at the design changes proposed by @danielHype. They take care of this and clean this process up. Just implement the modified layout from Figma.